PEN-786: Test coverage for default output block

beltrancaliz opened the pull request:
Lots of unit tests
[Jira ticket](https://arcpublishing.atlassian.net/browse/PEN-786)
![image](https://user-images.githubusercontent.com/21249916/78044495-618ab880-733a-11ea-8f5f-efe17c792e50.png)

beltrancaliz committed:
Default output type basic test

beltrancaliz committed:
Unit tests for default output block and article page

beltrancaliz committed:
video pagetype unit tests

beltrancaliz committed:
gallery pagetype unit tests

beltrancaliz committed:
Unit tests for author page type

beltrancaliz committed:
Unit tests for tag page type

beltrancaliz committed:
Unit tests for twitter meta tags

beltrancaliz committed:
Unit tests for author pages with gc and custom tags

beltrancaliz committed:
Unit tests for tag pages with gc and custom tags

beltrancaliz committed:
Unit tests for article pages with gc and custom tags

beltrancaliz committed:
Unit tests for video pages with gc and custom tags

beltrancaliz committed:
Unit tests for gallery pages with gc and custom tags

beltrancaliz committed:
Unit tests to cover tags instead of keywords

JackHowa reviewed (approved):
Boom -- great work on this. Hope it wasn't too painful. Thanks

brentmiller1973 reviewed (approved):
Nice Work!

beltrancaliz committed:
Merge branch 'master' into metadata-unit-tests

beltrancaliz committed:
Publish

 - @wpmedia/article-body-block@2.0.9
 - @wpmedia/article-tag-block@2.0.9
 - @wpmedia/author-bio-block@1.0.9
 - @wpmedia/author-content-source-block@1.0.9
 - @wpmedia/byline-block@1.0.9
 - @wpmedia/card-list-block@1.0.9
 - @wpmedia/collections-content-source-block@1.1.1
 - @wpmedia/content-api-source-block@1.0.9
 - @wpmedia/date-block@2.0.9
 - @wpmedia/default-output-block@2.2.0
 - @wpmedia/double-chain-block@1.0.9
 - @wpmedia/extra-large-manual-promo-block@1.0.9
 - @wpmedia/extra-large-promo-block@1.0.10
 - @wpmedia/footer-block@2.0.9
 - @wpmedia/full-author-bio-block@1.0.9
 - @wpmedia/gallery-block@1.0.9
 - @wpmedia/header-nav-block@2.0.9
 - @wpmedia/headline-block@2.0.9
 - @wpmedia/htmlbox-block@1.0.9
 - @wpmedia/large-manual-promo-block@1.0.9
 - @wpmedia/large-promo-block@1.0.10
 - @wpmedia/lead-art-block@2.0.9
 - @wpmedia/links-bar-block@2.0.9
 - @wpmedia/masthead-block@1.0.9
 - @wpmedia/medium-manual-promo-block@1.0.9
 - @wpmedia/medium-promo-block@1.0.10
 - @wpmedia/numbered-list-block@3.0.9
 - @wpmedia/overline-block@1.0.9
 - @wpmedia/quad-chain-block@1.0.4
 - @wpmedia/results-list-block@2.0.9
 - @wpmedia/right-rail-block@1.0.9
 - @wpmedia/search-content-source-block@1.0.9
 - @wpmedia/search-results-list-block@0.2.4
 - @wpmedia/section-title-block@1.1.4
 - @wpmedia/share-bar-block@2.0.9
 - @wpmedia/shared-styles@1.0.9
 - @wpmedia/simple-list-block@2.0.9
 - @wpmedia/single-chain-block@1.0.9
 - @wpmedia/site-hierarchy-content-block@3.1.8
 - @wpmedia/small-manual-promo-block@1.0.9
 - @wpmedia/small-promo-block@1.0.10
 - @wpmedia/story-feed-author-content-source-block@1.0.9
 - @wpmedia/story-feed-query-content-source-block@1.0.9
 - @wpmedia/story-feed-sections-content-source-block@1.0.9
 - @wpmedia/story-feed-tag-content-source-block@1.0.9
 - @wpmedia/subheadline-block@2.0.9
 - @wpmedia/tag-title-block@1.0.9
 - @wpmedia/tags-content-source-block@1.0.9
 - @wpmedia/top-table-list-block@1.0.9
 - @wpmedia/triple-chain-block@1.0.9
 - @wpmedia/unpublished-content-source-block@1.0.9
 - @wpmedia/video-player-block@1.0.9
